src/v/redpanda/application.cc

```diff
@@ -429,7 +481,32 @@ void application::wire_up_services() {
                            .get0();
     kafka_cfg.credentials
       = kafka_builder
-          ? kafka_builder->build_reloadable_server_credentials().get0()
+          ? kafka_builder
+              ->build_reloadable_server_credentials(
+                [](
```

Comment:
can we factor out this block--it looks like it is repeated at least a few times and can be parameterized on the logger and maybe a sstring name for context.

Reply:
done